FEXCore/Source/Interface/Core/Core.cpp

```diff
@@ -227,19 +227,70 @@ namespace FEXCore::Context {
 
     // Currently these flags just map 1:1 inside of the resulting value.
     for (size_t i = 0; i < FEXCore::Core::CPUState::NUM_EFLAG_BITS; ++i) {
-      EFLAGS |= Frame->State.flags[i] << i;
+      switch (i) {
+        case X86State::RFLAG_PF_LOC:
+          // Intentionally do nothing.
+          // PF can contain up to 4-bits which can corrupt other members when compacted.
+        break;
+
+        default:
+          EFLAGS |= Frame->State.flags[i] << i;
```

Comment:
```suggestion
          EFLAGS |= uint32_t{Frame->State.flags[i]} << i;
```

since `i` can be `31`